slides/lectures/01/admin/slides.md

```diff
@@ -0,0 +1,36 @@
+---
+---
+# Outline
+
+<div grid="~ cols-2 gap-5">
+<div>
+
+## Lectures
+  - 12 lectures
+  - 1 Q&A lecture for the project
+
+## Labs
+  - 12 labs
+
+## Project
+- You have to build a project
+- Will cost around 150 RON
+- Presented at PM Fair during the last week of the semester
```

Comment:
What is PM Fair

Reply:
https://ocw.cs.pub.ro/courses/pm/pm/lab/regulament - project presentations at the booths, nu?

Reply:
The text is too much now, overflowing the screen.